### DISCUSSION TITLE
Your favorite Example/Demo

### DISCUSSION THREAD

**Discussion by lajbel** [replit/kaboom#184] [General]
Now that I've seen the [new demos](https://kaboom2000.slmjkdbtl.repl.co/demo) I don't know whether to decide on eatbomb or the shooter (from the previous page)

**Comment by slmjkdbtl** [comment]
Haha eatbomb is definitely the most action-packed. The shooter is still there I think? Temporarily removed the scrolling background tho.

**Reply by lajbel** [reply]
I am seriously thinking of making a remake of the original version of the shooter example, such a jewel cannot be abandoned

**Reply by slmjkdbtl** [reply]
What do you think makes the original one better? The sprites?

**Reply by lajbel** [reply]
The sprites, the background with stars, it is the example that comes closest to a full-fledged game. I also love music!

**Reply by slmjkdbtl** [reply]
Oh yeah they'll definitely be in the new one. The music is from Earthbound
